examples/simple-game-server/Makefile

```diff
@@ -76,15 +76,14 @@ endif
 # Builds all image artifacts and create a docker manifest that is used to inform the CRI (Docker or containerd usually)
 # which image is the best fit for the host. See https://www.docker.com/blog/multi-arch-images/ for details.
 push: push-linux-image-amd64
-	-DOCKER_CLI_EXPERIMENTAL=enabled docker manifest rm $(server_tag)
-	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create $(server_tag) $(push_server_manifest)
 ifeq ($(WITH_WINDOWS), 1)
 push: push-windows-images
-	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest annotate --os.version $(WINDOWS_VERSIONS) 	
 endif
 ifeq ($(WITH_ARM64), 1)
 push: push-linux-image-arm64
 endif
+	-DOCKER_CLI_EXPERIMENTAL=enabled docker manifest rm $(server_tag)
```

Comment:
@Ludea - you previously had `docker manifest annotate --os.version $(WINDOWS_VERSIONS)` - but I'm not sure what it does, and we didn't have it anywhere else, so it was just easier to remove it.

Does it do anything important 😁?

Reply:
Also, just so you know - I rearranged this, because the command between target declarations don't run -- you actually can't do a `if` conditional around commands in a makefile, you can only change top level declarations (like add more dependencies).

Reply:
> you previously had docker manifest annotate --os.version $(WINDOWS_VERSIONS) - but I'm not sure what it does, and we didn't have it anywhere else, so it was just easier to remove it.

I had it because of this comment https://github.com/googleforgames/agones/commit/cb7c28c08e45b37aa951973497c19c2809722354#diff-6f4dc5b6a00ecf8904eece1a3f411736a2ae6bb76457b93b92b2360f2099060dL64

Reply:
Aaah. I _think_ it's fine without it, but I guess we will see 😄